ftplugin/php/splitjoin.vim

```diff
@@ -1,6 +1,6 @@
 let b:splitjoin_split_callbacks = [
       \ 'sj#js#SplitArray',
-      \ 'sj#php#SplitArray',
+      \ 'sj#php#SplitBraces',
```

Comment:
Fair enough. I'll merge it in and see if swapping the functions will do the trick.